Formula/kyuubi.rb

```diff
@@ -0,0 +1,36 @@
+class Kyuubi < Formula
+  desc "Serverless SQL on data warehouses and lakehouses"
+  homepage "https://kyuubi.apache.org/"
+  version "1.6.1-incubating"
+  url "https://dlcdn.apache.org/kyuubi/kyuubi-#{version}/apache-kyuubi-#{version}-bin.tgz"
```

Comment:
let's maybe do in this way, consistent with the other apache formulae
```suggestion
  url "https://www.apache.org/dyn/closer.lua?path=kyuubi/1.6.1/apache-kyuubi-1.6.1-incubating-bin.tar.gz"
  mirror "https://archive.apache.org/dist/kyuubi/kyuubi-1.6.1-incubating/apache-kyuubi-1.6.1-incubating-bin.tgz"
  version "1.6.1"
```